src/raygui.h

```diff
@@ -2480,6 +2501,7 @@ int GuiTextBox(Rectangle bounds, char *text, int bufferSize, bool editMode)
             if (IsKeyPressed(KEY_LEFT) || (IsKeyDown(KEY_LEFT) && (autoCursorCooldownCounter > RAYGUI_TEXTBOX_AUTO_CURSOR_COOLDOWN)))
             {
                 autoCursorDelayCounter++;
+                blinkCursorFrameCounter = 0;
```

Comment:
setting it to 0 when left/right is pressed so that as they're moving the cursor it doesnt blink